src/azure-cli/requirements.py3.windows.txt

```diff
@@ -120,6 +120,7 @@ pycomposefile==0.0.30
 pycparser==2.19
 PyGithub==1.55
 PyJWT==2.4.0
+pymsalruntime==0.14.0
```

Comment:
MSAL 1.27.0 requires

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.27.0/setup.cfg#L68

```
    pymsalruntime>=0.13.2,<0.15; python_version>='3.6' and platform_system=='Windows'
```

If `pymsalruntime` 0.13.12 is already installed, `azdev setup -c` won't upgrade it to 0.14.0, so `pymsalruntime` is explicitly pinned to 0.14.0.

Meanwhile, as Azure CLI doesn't directly depend on `pymsalruntime`, `pymsalruntime` is not pinned in `setup.py` and any version allowed by MSAL is acceptable.